### PR TITLE
wal-g

### PR DESCRIPTION
**Pull request by jbronn**
* Package up [`wal-g`](https://github.com/wal-g/wal-g) 2.0.0 for PostgreSQL.
* Upgrade Go version to 1.18.4.
* Fix some image tagging and `Makefile` issues.

### CODE REVIEW

File: docker-compose.el7.yml [hunk 1]
```diff
@@ -224,6 +224,10 @@ x-rpmbuild:
     rubygem-libxml-ruby:
       image: rpmbuild-rubygem-libxml-ruby
       version: &rubygem_libxml_ruby_version 3.2.2-1
+    sbt:
+      image: rpmbuild-generic
+      version: &sbt_version 1.6.1-1
+      arch: noarch
```
Comment (hummeltech) [inline docker-compose.el7.yml:230]: Thanks for alphabetizing these!

**Review by hummeltech — APPROVED** [review]
Looking good!

**Comment by hummeltech** [comment]
@jbronn I just realized this is missing `provenance.*.md` entries.

**Comment by jbronn** [comment]
@hummeltech fixed in 43810b6 of #61.

**Pull request merged**